src/Setup.php

```diff
@@ -4,19 +4,30 @@
 
 class Setup
 {
-	public static function activate() 
+	private MeetingReminderCommand $meetingReminder;
```

Comment:
The indentation of this file is messed up. Be consistent with indents. Use either 4 spaces or tabs